src/build/build.ts

```diff
@@ -24,11 +25,6 @@ import {PrefetchTransform} from './prefetch';
 import {waitFor} from './streams';
 import {parsePreCacheConfig} from './sw-precache';
 
-// TODO(fks) 09-22-2016: Latest npm type declaration resolves to a non-module
```

Comment:
Yea I tried to clarify this in the comment (/w "consider writing a custom declaration in the `custom_typings/` folder"). Could have been more clear.